util/kvcache/simple_lru.go

```diff
@@ -35,6 +36,20 @@ type cacheEntry struct {
 	value Value
 }
 
+var (
+	// GlobalLRUMemUsageTracker tracks all the memory usage of SimpleLRUCache
+	GlobalLRUMemUsageTracker *memory.Tracker
+)
+
+const (
+	// ProfileName is the function name in heap profile
+	ProfileName = "github.com/pingcap/tidb/util/kvcache.(*SimpleLRUCache).Put"
```

Comment:
If the item in the cache is deleted by LRU replacement, or by cache invalidation, I suppose this memory release would be reflected in the profile of this function name?

Reply:
Right. The heap profile 's flat value showed the current heap usage referenced for the given function. If the cached is released, it would be reflected in the next scraping result.